src/elevenlabs/conversational_ai/conversation.py

```diff
@@ -76,6 +85,8 @@ def __init__(
         *,
         requires_auth: bool,
         audio_interface: AudioInterface,
+        config: Optional[ConversationConfig] = ConversationConfig(),
```

Comment:
Optional: This is not modified currently, but would be easy to introduce a bug later by doing so.

https://docs.python.org/3.4/tutorial/controlflow.html#default-argument-values:~:text=Important%20warning%3A%20The%20default%20value%20is%20evaluated%20only%20once.%20This%20makes%20a%20difference%20when%20the%20default%20is%20a%20mutable%20object%20such%20as%20a%20list%2C%20dictionary%2C%20or%20instances%20of%20most%20classes.

Consier `config: Optional[ConversationConfig] = None` and in the body `self.config = config or ConversationConfig()`.

Reply:
this is added